docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -159,6 +159,9 @@ include::{project-root}/docs/src/test/java/com/google/cloud/spring/autoconfigure
 
 By default, the `SimplePubSubMessageConverter` is used to convert payloads of type `byte[]`, `ByteString`, `ByteBuffer`, and `String` to Pub/Sub messages.
 
+===== Advanced Usage
```

Comment:
Maybe something like "SpEL expression-based message routing" for the header?

Reply:
Found the block in `spring-integration-pubsub` that is a better home for this, and it doesn't need a header anymore.